packages/app/src/settings/SettingsContainer.vue

```diff
@@ -23,6 +23,7 @@
       >
         <ExternalEditorSettings :gql="props.gql" />
         <ProxySettings :gql="props.gql" />
+        <NotificationSettings :gql="props.gql" />
```

Comment:
I think the max-height of the SettingsCard this is inside of should be increased so it does not scroll by default.